QuizApp/App/Networking/Models/QuizDifficultyNetworkModel.swift

```diff
@@ -0,0 +1,64 @@
+enum QuizDifficultyNetworkModel: String, Codable {
+
+    case easy = "EASY"
+    case normal = "NORMAL"
+    case hard = "HARD"
+
+}
+
+enum QuizDifficultyDataModel: String {
+
+    case easy = "EASY"
```

Comment:
Only your network model will need to have rawValues defined, in order to parse JSON. Other layers don't need it (maybe just UI layer does in case you want to use rawValue for title)